_data/simple-icons.json

```diff
@@ -8791,6 +8791,11 @@
                 "type": "MPL-2.0"
             }
         },
+        {
+            "title": "LibreTube",
+            "hex": "FFBBAA",
```

Comment:
I would go with `#FF9699`, using https://meyerweb.com/eric/tools/color-blend/#FF5A8B:FFD1A7:1:hex from https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines. That accent color specifically isn't present in the color logo, and I think this darker red is more fitting just as far as the eye goes.

Reply:
Agreed. I just updated the color hex code.